bin/git-jira-pr

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env ruby
+# encoding: UTF-8
+#
+# git-jira-pr --
+#
+#   Open a pull request for the current branch in your default browser
+#
+#   Assumes the branches are named
+#   <team>/<branch-title>-<story-id>
+#
+require 'rubygems'
+require 'jira'
+require 'optparse'
+require 'cgi'
+require 'term/ansicolor'
+require 'git-whistles/app'
+require 'git-whistles/jira'
+
+class App < Git::Whistles::App
```

Comment:
For the consistency do you mind changing this file to only have 1 empty line inbetween methods?